### DISCUSSION TITLE
kyuubi是否有付费版本？

### DISCUSSION THREAD

**Discussion by zhujian0614** [apache/kyuubi#4945] [General]
我们希望有一个类似hue的统一查询平台，支持审计、监控、权限管控、sql兼容、查询路由等功能，和kyuubi类似，能代理多个后端引擎

**Comment by yaooqinn** [comment]
据我所知，国内类似云上的话，阿里云，腾讯云都在 EMR 套件中封装了 Kyuubi 的。私有平台的话，网易有数的EasyData和NDH产品是提供的。国外的话 AWS， Cloudera 都有付费的服务帮助用户将 Kyuubi 集成到他们的 EMR或者 CDP(H) 平台中
